README.md

```diff
@@ -72,11 +72,11 @@ To make a completely open sourced in house hotel management system
 
 
 -   [Complete Intro to React, v5](https://frontendmasters.com/courses/complete-react-v5/)
-    
+
```

Comment:
you have added unnecessary spaces please remove them

Reply:
yeah sure